impeller/docs/faq.md

```diff
@@ -100,3 +100,133 @@
     as development progresses.
 * How do you run `impeller_unittests` with Playgrounds enabled?
   * Specify the `--enable_playground` command-line option.
+* Describe Impeller in a Tweet.
+  * "Impeller is AOT mode for rendering."
```

Comment:
spell out AOT. Suggest "Ahead of Time (AOT) mode ..."

Reply:
Shouldn't we also mention what is being AOT'ed? Like "Impeller is rendering with ahead-of-time (AOT) compilation"?

Reply:
Seths suggestion done. Michael, that should become clearer in the next entry. Should I add another FAQ entry perhaps?